### PR TITLE
Issue 832: creating run_id if not given while executing trigger_dag

### PR DESCRIPTION
**Pull request by msumit**

### CODE REVIEW

File: airflow/bin/cli.py [hunk 1]
```diff
@@ -84,14 +84,15 @@ def trigger_dag(args):
     session = settings.Session()
     # TODO: verify dag_id
     execution_date = datetime.now()
+    run_id = args.run_id or "manual__{0}".format(datetime.datetime.utcnow().isoformat())
```
Comment (mistercrunch) [inline airflow/bin/cli.py]: did you test this? datetime came from a `from datetime import datetime`, so datetime.datetime won't work. I think you should just use `execution_date.isoformat()`

Comment (msumit) [inline airflow/bin/cli.py]: Oops, my bad.. still a n00b in python.. will fix it..

Comment (msumit) [inline airflow/bin/cli.py]: btw just saw that we do import `datetime`.. https://github.com/airbnb/airflow/blob/master/airflow/bin/cli.py#L6

anyways will update the PR to use `execution` date..

**Comment by mistercrunch** [comment]
LGTM! thanks!

**Pull request merged**